### PR TITLE
Update user config overrides for v12

### PR DESCRIPTION
**Pull request by CarlosFdez**
The appearance of core side is different from pf2e side due to some unfortunate text wrapping of the label in standard-form mode. We might have to make some changes later on to form styles, or see if core bumps into the same issue as they continue to convert. We'll see!

image

image

### CODE REVIEW

File: src/module/user/sheet.ts [hunk 1]
```diff
@@ -1,14 +1,62 @@
...
+
+    override tabGroups = {
+        primary: "core",
+    };
+
+    #getTabs() {
+        const DEFAULTS = { group: "primary" as const, active: false, cssClass: "" };
+        const tabs = [
+            { ...DEFAULTS, id: "core", icon: "fa-solid fa-cog", label: "Core" },
+            { ...DEFAULTS, id: "pf2e", icon: "fa-solid fa-cog", label: "PF2E.Pathfinder" },
```
Comment (In3luki) [inline src/module/user/sheet.ts:26]: I don't know if `Pathfinder` is the best label for this. `Pathfinder Second Edition` like in the settings is probably too long. Maybe `System`?

File: types/foundry/build/generate-client-base-mixes.ts [hunk 2]
```diff
@@ -470,6 +470,12 @@ const clientDocs: Record<string, { hasSheet?: boolean; isCanvasDoc?: boolean; pa
     Wall: { isCanvasDoc: true },
 };
 
+// imports
+console.log(
+    String.raw`import type { ApplicationV2, DocumentSheetV2 } from "../../../client-esm/applications/api/module.d.ts";
+
+`,
+);
```
Comment (In3luki) [inline types/foundry/build/generate-client-base-mixes.ts:478]: I have no idead how this works so it's probably not an error, just checking that this is intended.

Comment (In3luki) [reply types/foundry/build/generate-client-base-mixes.ts:478]: I've looked at the rest of the file and this is intended after all.

File: src/module/user/sheet.ts (hunk 1 above)
Comment (CarlosFdez) [reply src/module/user/sheet.ts:26]: Sure. If you got any idea for what would be good font awesome choices as well I'm all ears.

Comment (In3luki) [reply src/module/user/sheet.ts]: `fa-user` for the first tab and maybe `fa-dice` (like the systems tab in the setup screen) for the other?

Comment (CarlosFdez) [reply src/module/user/sheet.ts]: Done! Good choice.

**Pull request merged**